tests/integration/pkg/tests/devfilelibrary_tests.go

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (c) 2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package tests
+
+import (
+	"os"
+	"path"
+
+	devfilePkg "github.com/devfile/library/pkg/devfile"
+	"github.com/devfile/library/pkg/devfile/parser"
+	"github.com/devfile/registry-support/tests/integration/pkg/util"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+)
+
+// Integration/e2e test logic based on https://github.com/devfile/registry-operator/tree/master/test/integration
+// Tests use the CLI version of the registry-library to test.
+// Note: Requires adding the CLI to the system path before running these tests
+var _ = ginkgo.Describe("[Verify registry library works with registry]", func() {
```

Comment:
We can also test the case where listing both stack and sample, which to test `registry-library list --type all`.

Reply:
Thanks, good catch. Updated.